ndk-glue/src/lib.rs

```diff
@@ -122,8 +122,16 @@ pub enum Event {
     WindowCreated,
```

Comment:
Should we add some docs about acquiring and holding the lock here too? And on `InputQueueCreated`?